src/template.rs

```diff
@@ -136,7 +136,7 @@ pub struct HelperTemplate {
     pub template: Option<Template>,
     pub inverse: Option<Template>,
     pub block: bool,
-    pub chain: bool,
+    chain: bool,
```

Comment:
Wasn't this a breaking change?

Before: https://docs.rs/handlebars/5.1.0/handlebars/template/struct.HelperTemplate.html
After: https://docs.rs/handlebars/latest/handlebars/template/struct.HelperTemplate.html

Both removes access to a field and makes the struct no longer directly initializable.  If the latter is an expected property, this should have `#[non_exhaustive]` (but that too is a breaking change)

Reply:
I see. I didn't realized initialization issue with introducing a private field. Will revert this and yank 5.1.1 for new solution.

Reply:
This isn't just about initialization but also that a release already exists with `pub chain`

Reply:
This field is introduced in 5.1.0 and not used in any API. The impact should be minimised.